.travis.yml

```diff
@@ -0,0 +1,29 @@
+dist: trusty
+language: python
+python:
+- 2.7
+- 3.5
+- 3.6
+addons:
+  apt:
+    packages:
+    - libhdf5-dev
+cache: pip
+install:
+- pip install tox
+- pip install .
+script:
+  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then tox -e py27; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then tox --skip-missing-interpreters; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then tox -e py36; fi
+
+before_deploy:
+- pip install twine
+deploy:
+  provider: pypi
+  user: wizmer
```

Comment:
this shouldn't be your user account, needs to be something BBP

Reply:
I know, it's supposed to be bbp.opensource but I did not manage to register the encrypted password. I need to create a ticket to infra I think.